packages/system-upgrade-controller/charts/values.yaml

```diff
@@ -0,0 +1,13 @@
+global:
+  cattle:
+    systemDefaultRegistry: ""
+
+systemUpgradeController:
+  image:
+    repository: rancher/system-upgrade-controller
+    tag: v0.14.2
+
+kubectl:
+  image:
+    repository: rancher/kubectl
+    tag: v1.31.1
```

Comment:
That's quite a jump from 1.23.3 we've used for a good while! Should be fine and actually help with CVEs and such.